<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSignalDialog</class>
 <widget class="QWidget" name="NewSignalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButtonSin">
          <property name="text">
           <string>Sin</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonCos">
          <property name="text">
           <string>Cos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="PhaseLabel">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="horizontalSliderPhase">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPhase">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>33</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButtonPlot">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="AmplitudeLabel">
       <property name="text">
        <string>Amplitude</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="horizontalSliderAmplitude">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAmplitude">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>33</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="FrequencyLabel">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="horizontalSliderFrequency">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxFrequency">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>33</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderAmplitude</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxAmplitude</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxAmplitude</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderAmplitude</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFrequency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxFrequency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFrequency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderFrequency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderPhase</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxPhase</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPhase</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderPhase</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
